server/helpers/tripHelper.js

```diff
@@ -0,0 +1,161 @@
+import models from '../sequelize/models';
+
+const {
+  Trip,
+  User
+} = models;
+
+/**
+ * This class contains
+ * all methods required to save/edit/retrieve
+ * the trip data
+ */
+class TripHelpers {
+  /**
+     * Finds a trip by reasons and date.
+     * @param {string} trip a trip data.
+     * @returns {object} trip data.
+     */
+  static async reasonsDate(trip) {
```

Comment:
Change the method to **findByReasonOrDate** instead

Reply:
Feedback implemented